pythran/spec.py

```diff
@@ -316,5 +319,5 @@ def specs_to_docstrings(specs, docstrings):
         )
 
 
-def spec_parser(path):
-    return SpecParser()(path)
+def spec_parser(path, python_file=True):
```

Comment:
I would feel more comfortable with a subclass of SpecParser that just overrides the right method, putting shared code where it's meant to be, instead of this ugly boolean